lib/newark/app.rb

```diff
@@ -72,9 +76,18 @@ def params
       request.params
     end
 
+    def route_id
+      request.env[ROUTE_ID]
+    end
+
+    def set_route_id(id)
```

Comment:
Setter methods in an included module requires the use of `self`, ie `self.route_id = 'my route'`, otherwise, a local instance variable is assigned.  It's just a preference to set the route id with `set_route_id('route')` instead of `self.route_id = 'route'`, so I'm fine with changing it.